### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Reshmika-Nambiar**
I have updated the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+* Devayani Shivankar
+* Vaishnavi Jarwal
+* Navya Sharma
+* Reshmika Sreenath Nambiar
+
+## Mentor
+* Shubhankar Gupta
+
+## Description
+RunAway is an endless runner Android game built on Unity. The player controls a mermaid who turns into a human on land. They have to dodge obstacles and collect coins to
```
Comment (DevayaniShivankar) [inline README.md]: The mermaid scene hasn't been added yet, so mention that as a future scope, or an upcoming feature.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+the longest distance run, or the total coins collected.
+
+* GitHub repo link: [Link to repository](https://github.com/DevayaniShivankar/RunAway)
+* Drive link: [Drive link here](https://drive.google.com/)
+
+## Technology stack
+
+1. C#
+2. Unity
+3. Firebase
```
Comment (DevayaniShivankar) [inline README.md]: Mention these points as:
1. Scripting Language: C#
2. Platform: Unity (2020.3.3f1)
3. Database: Firebase

**Review by DevayaniShivankar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Reshmika-Nambiar) [reply README.md]: I've added this now.

File: README.md (hunk 1 above)
Comment (Reshmika-Nambiar) [reply README.md]: I have put this part in the Future Scope.

**Comment by Reshmika-Nambiar** [comment]
I have made the proposed changes. Please review it once more.

**Review by DevayaniShivankar — APPROVED** [review]
It is all good so far, will add more to it soon.

**Pull request merged**